examples/appengine/hello.go

```diff
@@ -2,10 +2,10 @@ package mellow
 
 import (
 	"fmt"
-	"github.com/bugsnag/bugsnag-go"
-	"github.com/bugsnag/bugsnag-go/errors"
 	"net/http"
 	"os"
+
+	"github.com/bugsnag/bugsnag-go"
```

Comment:
I use this with my editor https://godoc.org/golang.org/x/tools/cmd/goimports. I did a quick verify that `go fmt` didn't change the import order though, using `go1.5.1`